azext_iot/tests/test_iot_ext_int.py

```diff
@@ -1437,9 +1437,51 @@ def test_storage(self):
         )
 
     @pytest.mark.skipif(
-        not LIVE_IDENTITY, reason="azext_iot_testidentity env var not set"
+        not LIVE_STORAGE_ID or not LIVE_STORAGE,
+        reason="azext_iot_identity_teststorageid and azext_iot_teststorageuri env vars not set",
     )
     def test_identity_storage(self):
+        identity_type_enable = "SystemAssigned"
+        identity_type_disable = "None"
+        storage_role = "Storage Blob Data Contributor"
+
+        # check hub identity
+        identity_enabled = True
```

Comment:
Should this start as `False`? Otherwise the role assignment clean-up will always fire.

Reply:
Yes, it totally should - good catch.